content/_index.md

```diff
@@ -7,10 +7,26 @@ BookToC: false
 
 # The Linux Userspace API (UAPI) Group
 
-The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, and run modern Linux operating systems.
+The userspace API ("uapi") group is a cross-distro community for people with an interest in innovating how we build, deploy, run, and safely update modern Linux operating systems.
 It serves as a central gathering place for specs, documentation, and ideas.
 
-Follow [@uapi-group](https://twitter.com/uapi_group) on twitter to catch the latest news!
+In scope of the group are secure boot mechanisms, kernel / disk image formats, TPM usage, safe and secure updates, and related topics.
```

Comment:
>secure boot mechanisms

Do you mean "Secure Boot" as the technology or the general concept of securing the bootchain? It might be better to call it "trustworthy boot mechanisms" or "trustable boot mechanisms" I think?